src/actions/keys.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const alt = require('../alt');
+
+class KeyActions {
+  findNext(evt) {
+    this.dispatch(evt);
+  }
+  findPrevious(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollUpLine(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollDownLine(evt) {
+    this.dispatch(evt);
+  }
+  tab(evt) {
```

Comment:
No, just because they all relate to the editor doesn't make them logically the same actions.